test/ui/browser/actions/test-duplicate-tab.js

```diff
@@ -0,0 +1,28 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import expect from 'expect';
+import configureStore from '../../../../ui/browser/store/store';
+import * as actions from '../../../../ui/browser/actions/main-actions';
+
+describe('Action - DUPLICATE_TAB', () => {
+  beforeEach(function() {
+    this.store = configureStore();
+  });
+
+  it('Should create a new tab with source\'s location and select it', function() {
+    const { getState, dispatch } = this.store;
+    expect(getState().currentPageIndex).toEqual(0);
+
+    dispatch(actions.createTab('https://moz.org/1'));
+    dispatch(actions.createTab('https://moz.org/2'));
+    dispatch(actions.createTab('https://moz.org/3'));
+
+    dispatch(actions.duplicateTab(1));
+    expect(getState().pages.size).toEqual(5);
+    expect(getState().pages.get(4).location).toEqual('https://moz.org/1',
+      'DUPLICATE_TAB creates an appends a new tab with source\'s location');
```

Comment:
Do we have to use single quotes even when there's a single quote in the string or is eslint smart enough to not complain about this?